src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -182,5 +209,37 @@ private bool IsAttributeDerivedFromClassInternal(ITypeSymbol attributeType, ITyp
 
 		private bool IsAggregatorAttribute(ITypeSymbol attributeType) =>
 			attributeType.InheritsFromOrEquals(_aggregateAttribute) || attributeType.InheritsFromOrEquals(_dynamicAggregateAttribute);
+
+
+		///TODO: refactoring arguments -> remove semanticModel to constructor? Is it nessesary?
+		///TODO: Add DataFlow analyze to corner cases with defaul IsDBBound assigment.
+		public bool? IsBoundField(PropertyDeclarationSyntax property, SemanticModel semanticModel)
+		{
+			var typeSymbol = semanticModel.GetDeclaredSymbol(property);
+			typeSymbol.ThrowOnNull();
+
+			var attributesData = typeSymbol.GetAttributes();
+
+			foreach (var attribute in attributesData)
+			{
+				if (IsBoundAttribute(attribute) == BoundType.DbBound)
+					return true;
+
+				foreach (var argument in attribute.NamedArguments)
+				{
+					if (argument.Key == IsDBField && !argument.Value.IsNull && argument.Value.Value is bool boolValue && boolValue == true)
+						return boolValue;
+				}
+			}
+			return false;
+		}
+
+	}
+
+	public enum BoundType
```

Comment:
@saloponov the enum is better to be placed in the separate file or at least at the start of the file